tests/sku_create_test.py

```diff
@@ -49,6 +50,9 @@ def are_file_contents_same(self,fname1,fname2):
         return True
     
     def test_no_param(self):
+        if (os.path.exists(output_dir_path)):
+            shutil.rmtree(output_dir_path)
```

Comment:
Do you need to `import shutil`?

Reply:
Yes, I have added it in the next commit.